### PR TITLE
AmazonS3Client unit test using LocalStackContainer

### PR DESCRIPTION
**Pull request by dharmverma**
AmazonS3Client unit test using LocalStackContainer

### CODE REVIEW

**Review by JBaigGoaco — APPROVED** [review]

File: Libraries/CO.CDP.AwsServices.Tests/AwsFileManagerTest.cs [hunk 1]
```diff
@@ -65,32 +66,19 @@ public async Task UploadFile_UploadsFileUsingPutObject_WhenFileIsSmall()
...
 
         await _fileManager.UploadFile(stream, filename);
-
-        _mockS3Client.Verify(s3 => s3.PutObjectAsync(It.IsAny<PutObjectRequest>(), default), Times.Never);
-        _mockTransferUtility.Verify(s3 => s3.UploadAsync(It.Is<Amazon.S3.Transfer.TransferUtilityUploadRequest>(req =>
-            req.BucketName == StagingBucket &&
-            req.Key == filename &&
-            req.InputStream == stream &&
-            req.ContentType == "application/octet-stream"
-        ), default), Times.Once);
```
Comment (jakzal) [inline Libraries/CO.CDP.AwsServices.Tests/AwsFileManagerTest.cs:93]: Is there a way to assert if the file was uploaded? We're left with tests that make no assertions.

Comment (jakzal) [reply Libraries/CO.CDP.AwsServices.Tests/AwsFileManagerTest.cs:93]: Perhaps, inspecting that `await _fileManager.UploadFile(stream, filename);` result is successful?

Comment (jakzal) [reply Libraries/CO.CDP.AwsServices.Tests/AwsFileManagerTest.cs:93]: Above applies to other tests .

Comment (dharmverma) [reply Libraries/CO.CDP.AwsServices.Tests/AwsFileManagerTest.cs:93]: Very valid point. there is code to validate it now.

**Review by jakzal — APPROVED** [review]

**Pull request merged**